### DISCUSSION TITLE
获取的票务信息和b站官方api给出的不一致（v2.8.9p1）

### DISCUSSION THREAD

**Discussion by HsuJv** [mikumifa/biliTickerBuy#422] [提问专区] [answered]
如图，通过网页端b站的api获取到的票价是98元一张
![image](https://github.com/user-attachments/assets/c69558b6-09ac-4c53-9c65-5c1505f36a0a)
![image](https://github.com/user-attachments/assets/6d32e61a-ea76-43b1-8f6a-f6e471c5eb37)

但是程序获取的票务信息显示97.99，最后生成的购买配置里的   "pay_money" 也是9799，需要手动改成9800才能购买
![image](https://github.com/user-attachments/assets/c95fd411-50d8-4a69-92b4-a543c986d662)
![image](https://github.com/user-attachments/assets/1c53006d-24e5-4daa-a858-e4dfed307af2)

**Comment by mikumifa** [comment, marked as answer]
最新一版应该fix了

**Comment by mikumifa** [comment]
b站邮费计算的接口逻辑改了，现在邮费可能是复数，表示到付或者免邮费
